Makefile

```diff
@@ -26,7 +27,8 @@ drafter: config.gypi $(BUILD_DIR)/Makefile
 	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/$@ ./bin/$@
 
 install: drafter
-	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(DESTDIR)/drafter
+	mkdir -p $(BINDIR)
+	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(BINDIR)/drafter
```

Comment:
I disagree ... at least in the Linux world you expect the headers and libs go to `/usr/local/include` and `/usr/local/lib` respectively, I think that there should be a `install_lib` target doing exactly this.